lib/nostrum/api.ex

```diff
@@ -4462,62 +2142,35 @@ defmodule Nostrum.Api do
   end
 
   # If `:embed` is present, prepend to `:embeds` for compatibility
-  defp combine_embeds(%{embed: embed} = args),
+  def combine_embeds(%{embed: embed} = args),
     do: Map.delete(args, :embed) |> Map.put(:embeds, [embed | args[:embeds] || []])
 
-  defp combine_embeds(%{data: data} = args), do: %{args | data: combine_embeds(data)}
-  defp combine_embeds(%{message: data} = args), do: %{args | message: combine_embeds(data)}
-  defp combine_embeds(args), do: args
+  def combine_embeds(%{data: data} = args), do: %{args | data: combine_embeds(data)}
+  def combine_embeds(%{message: data} = args), do: %{args | message: combine_embeds(data)}
+  def combine_embeds(args), do: args
```

Comment:
Can you `@doc false` all the functions in this module you had to make public or move them into `Nostrum.Api.Helpers`?

Reply:
Yep. Moved them over to `Helpers` 👍